src/handlers/VehicleTestController.ts

```diff
@@ -1,58 +1,230 @@
 import * as enums from "../assets/Enums";
+import * as utils from "../utils";
+import * as models from "../models";
+
 import { IVehicleTestController } from "./IVehicleTestController";
 import { IExpiryDateStrategy } from "./expiry/IExpiryDateStrategy";
-import {
-  ExpiryDateStrategyFactory
-} from "./expiry/ExpiryDateStrategyFactory";
+import { ExpiryDateStrategyFactory } from "./expiry/ExpiryDateStrategyFactory";
 import { TestTypeForExpiry } from "../models/TestTypeforExpiry";
 import { Service } from "../models/injector/ServiceDecorator";
 import { TestDataProvider } from "./expiry/providers/TestDataProvider";
 import { DateProvider } from "./expiry/providers/DateProvider";
-import { ValidationUtil } from "../utils";
-import * as models from "../models";
+import { TestTypeParams } from "../models/TestTypeParams";
```

Comment:
I think it is fixed on the first PR but that could be a default import or we could be using a barrel file.

Reply:
removed unused import